src/components/ButtonPanel.js

```diff
@@ -0,0 +1,39 @@
+const ButtonPanel = () => (
+  <main>
+    <div>
+      <button name="symbolReset" type="submit"> AC </button>
```

Comment:
We are not going to use a button tag here, you already created a component with the button tag which is `Button`, so import `Button` from `Button.js` and use it.

And also, you need to divide buttons into groups;
![image](https://user-images.githubusercontent.com/31897434/118619881-e5149c80-b7bc-11eb-9961-a24332d8a5ff.png)